kortex_driver/src/hardware_interface.cpp

```diff
@@ -76,7 +76,7 @@ KortexMultiInterfaceHardware::KortexMultiInterfaceHardware()
   use_internal_bus_gripper_comm_(false)
 {
   RCLCPP_INFO(LOGGER, "Setting severity threshold to DEBUG");
-  auto ret = rcutils_logging_set_logger_level(LOGGER.get_name(), RCUTILS_LOG_SEVERITY_DEBUG);
+  auto ret = rcutils_logging_set_logger_level(LOGGER.get_name(), RCUTILS_LOG_SEVERITY_ERROR);
```

Comment:
Would we want to set this to a Info or Warn level?

Reply:
Info has some useful stuff but not too much logging, I can set it to that